website/events/api/v2/admin/serializers/event.py

```diff
@@ -30,17 +28,13 @@ def to_internal_value(self, data):
         self.fields["organisers"] = serializers.PrimaryKeyRelatedField(
             queryset=MemberGroup.active_objects.all()
         )
-        self.fields["slide"] = serializers.PrimaryKeyRelatedField(
-            queryset=Slide.objects.all()
```

Comment:
Do we want to remove slides from the API? Or do we still want them, by fetching them from the onetoonekey?

Reply:
I would propose removing them from the API if it doesn't break stuff